dist/components/MapLibreMap/lib/MapLibreGlWrapper.d.ts

```diff
@@ -1,5 +1,16 @@
 import { IControl, MapOptions as MapOptionsType } from "!maplibre-gl";
 import { Map as MapType } from "maplibre-gl";
+import maplibreGl from "maplibre-gl";

```

Comment:
please remove all changes of this file. All files under the ./dist/ folder are autogenerated during the build process (`yarn build`). No manual changes required.